UI/src/model/selectedLaneModelling.js

```diff
@@ -382,7 +387,7 @@
 
       //expiredLane could be modified by the user so we need to fetch the original
       var originalExpiredLane = _.find(lanesFetched, {'id': expiredLane.id});
-      if (linksSelected.length > 1) {
+      if (linksSelected.length > 1 && marker === undefined) {
```

Comment:
https://stackabuse.com/javascript-check-if-variable-is-a-undefined-or-null/ 
lodash löytyy metodi tohon undefine tarkistukseen mut ei nyt akuuttia